packages/ui/src/components/theme/styleOverrides/utils/shouldForwardProp.ts

```diff
@@ -0,0 +1 @@
+export const shouldForwardProp = (propName: PropertyKey, propsToSkip: PropertyKey[]) => !propsToSkip.includes(propName);
```

Comment:
if we **_can_** make `propsToSkip` an object of shape `Record<PropertyKey, boolean>` it would be more efficient to check if `PropertyKey` should be skipped and shorter in syntax so we won't need this function